### DISCUSSION TITLE
weird identation on pasting code

### DISCUSSION THREAD

**Discussion by abhinavpatel0** [helix-editor/helix#10820] [General] [answered]
### Summary

i got weird identation whenever i paste some code 
![image](https://github.com/helix-editor/helix/assets/121202898/e818c468-29d6-4b9b-9b79-d84e274e15bd)

### Reproduction Steps

 

i just compiled helix from source in windows 10 and every env variable is set up correctly

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

windows

### Terminal Emulator

https://github.com/microsoft/terminal

### Installation Method

source code

### Helix Version

helix 24.3 (f1c9580e)

**Comment by kirawi** [comment, marked as answer]
As was said on Matrix, use `space-p` instead of `ctrl-v`.
